docs/docs/FAQ.md

```diff
@@ -1,30 +1,30 @@
-# Frequently Asked Questions (FAQ)
+# bolt.diy Frequently Asked Questions (FAQ)
```

Comment:
remove bolt.diy, because this will become the page header, and each header will have bolt.diy at the beginning. which is not required

Reply:
removed